src/main/java/com/revlearn/team1/model/Course.java

```diff
@@ -43,9 +43,6 @@ public class Course {
     @OneToMany(mappedBy = "course", cascade = CascadeType.ALL)
     private List<DiscussionPost> discussionPosts = new ArrayList<>();
 
```

Comment:
wait also remove the changes to this file from your commit this will erase ralph's work lol